README.md

```diff
@@ -13,3 +13,34 @@ This is *not* a configuration management system, but is a handbook for operating
 A 'Meta-service' is an internal definition for 'things that involve lots of services working together' that makes it slightly less confusing to document at a high level... 
 
 This includes things like environmental monitoring, 
+

```

Comment:
Relative linking plox!

Reply:
TIL Relative Linking is a thing (regardless of how obvious the fact...)

Updated now.